calabash/Classes/FranklyServer/Routes/LPScreenshotRoute.m

```diff
@@ -32,7 +32,14 @@ - (BOOL) supportsMethod:(NSString *) method atPath:(NSString *) path {
 
 - (NSData *) takeScreenshot {
 
-  CGSize imageSize = [[UIScreen mainScreen] bounds].size;
+
+  // Available on iOS >= 7
+  SEL selector = @selector(drawViewHierarchyInRect:afterScreenUpdates:);
+  UIView *view = [[UIView alloc] initWithFrame:CGRectZero];
+  BOOL drawViewHierarchy = [view respondsToSelector:selector];
```

Comment:
Seems aight. If a UIView responds, UIWindow should ( I mean, it might not if they're doing something goofy with invocation forwards ) but I don't see why it wouldn't. 

<img width="249" alt="screen shot 2016-03-02 at 8 45 26 am" src="https://cloud.githubusercontent.com/assets/3009852/13467450/27c5016a-e053-11e5-8ef3-ba88720190ee.png">

That said, if you're eventually calling `drawViewHierarchyInRect:afterScreenUpdates:` on the window, why not check on the window itself? That's all that matters, right?